src/annotator/integrations/pdf.js

```diff
@@ -75,12 +80,11 @@ export class PDFIntegration extends TinyEmitter {
 
     this.annotator = annotator;
 
-    const window_ = /** @type {HypothesisWindow} */ (window);
-
     // Assume this class is only used if we're in the PDF.js viewer.
-    const pdfViewerApp = /** @type {PDFViewerApplication} */ (
-      window_.PDFViewerApplication
+    const pdfWindow = /** @type {PDFWindow} */ (
+      /** @type {unknown} */ (window)
```

Comment:
The double-cast here (`window => unknown => PDFWindow`) avoids an error message from TS, which specifically suggests this solution if we're sure of what we're doing.